extension/src/popup/components/sendPayment/styles.scss

```diff
@@ -75,6 +101,10 @@
     width: 100%;
     transition: opacity 0.15s ease-in-out;
 
+    &__full-height {
+      margin-top: 2rem !important;
```

Comment:
jc - why is `important` needed here? Often I find if I have to use this, something can be improved to not need it

Reply:
it's not 😄 . removed
https://github.com/stellar/freighter/pull/515/commits/fac9066a12c0871f45222808d13e29bd83c16349